### DISCUSSION TITLE
Results/reports watcher utility

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD-flow-scripts#985] [Ideas]
I find myself navigating to look at recent reports and results in reports/, logs/ and results/.

I was thinking it would be nice with a watcher GUI utility that just showed me a menu that I could click on to watch some recent development in results/, reports/ & logs/, perhaps kick off some new jobs.

For instance, if I have a flow going, it would be nice to see a progress bar, the floorplan of macros being showed (before any PDN or anything is generated, so it's quick to render), etc. I'm thinking this GUI should only do very light weight stuff from a CPU point of view, no waiting, just show what's going on.

Written in Python or something so that it is easy for the community to contribute. OpenROAD GUI has a very, very steep learning curve. This utility would be more to just use what is there to display progress and launch/stop actions.

A little bit of bling to OpenROAD...

**Comment by maliberty** [comment]
I proposed a flow manager a while ago. I hoped to get a gsoc student for it but no luck. It's not high enough priority to get attention but I'm glad to work with anyone who wants to contribute
